platforms/platforms.json

```diff
@@ -1,4 +1,5 @@
 {
+    "ios": "3.9.2"
```

Comment:
Missing comma, this is a syntax error. Also an outdated version, 4.1.1 is current, you should run "cordova platform update ios"